lua/lls.lua

```diff
@@ -16,6 +16,10 @@ return function (net_conf, numa_table)
 	local log_ratelimit_burst = 10
 	local front_max_pkt_burst = 32
 	local back_max_pkt_burst = 32
+	local front_icmp_msgs_per_sec = 1000
+	local front_icmp_msgs_burst = 50
+	local back_icmp_msgs_per_sec = 1000
+	local back_icmp_msgs_burst = 50
```

Comment:
Where did you get these values?

Reply:
The values follow the same configuration from GK blocks, i.e., ``lua/gk.lua``